components/LinkGrid/src/index.scss

```diff
@@ -0,0 +1,183 @@
+.denhaag-link-grid {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  overflow-x: hidden;
+  background-color: var(--denhaag-link-grid-panel-background-color);
+  padding-block-start: var(--denhaag-link-grid-panel-padding-sm);
+  padding-block-end: var(--denhaag-link-grid-panel-padding-sm);
```

Comment:
`padding-block-end: var(--denhaag-link-grid-panel-padding-sm);` de variabelnaam zoals hierboven toegelicht aanpassen naar `padding-block-end: var(--denhaag-link-grid-padding-block-end-sm);`

Reply:
Hey Robbie, 

De padding top en padding bottom is hier gelijk, maar ik mocht van de linter geen padding-block gebruiken.
Als ik het helemaal ga uitschrijven is dat niet helemaal DRY, of moeten die tokens altijd precies dezelfde benaming hebben als de css attributes?

Reply:
Done in de ge-update PR!